views/partials/social-media-buttons.ejs

```diff
@@ -0,0 +1,79 @@
+<div class="media-buttons-div">
+  <!-- Sharingbutton Facebook -->
+  <a
+    class="resp-sharing-button__link"
+    href="https://facebook.com/sharer/sharer.php?u=http%3A%2F%2Flocalhost%3A3000%2Fresults%2F<%-sessionId%>"
```

Comment:
Let's leave a comment here with TODO, to change links from localhost to the domain you will be using when hosting

Reply:
And the URL is broken somewhere

Reply:
The url actually works, it only appears broken because of directs to localhost, which facebook doesnt seem to recognize like a proper link. I tried changing this to an example link to an actual website, and it worked just fine. So this should work as well once the the page is actually up.